src/Sylius/Behat/Page/Shop/Product/ShowPage.php

```diff
@@ -121,6 +122,18 @@ public function getCatalogPromotionName(): string
         return explode(' - ', $this->getElement('catalog_promotion')->getText())[0];
     }
 
+    public function hasCatalogPromotionApplied(string $name): bool
+    {
+        $catalogPromotions = $this->getDocument()->findAll('css', '#promotion_label');
```

Comment:
I know that we already have this `id` but IIRC on-page we should only have single `id`